specialist-publisher/Dockerfile

```diff
@@ -0,0 +1,20 @@
+FROM ruby:2.6.3
+
+RUN apt-get update -qq && apt-get upgrade -y && apt-get install -y vim nodejs && apt-get clean
```

Comment:
Just noticed we're installing nodejs twice here.

Reply:
We're also doing 'clean' twice, but it's pretty screwed up in all the Dockerfiles, so not really part of this PR.

Reply:
I can fix that!